src/main/java/ecdar/backend/QueryHandler.java

```diff
@@ -162,9 +163,16 @@ private void handleQueryResponse(QueryProtos.QueryResponse value, Query query) {
                     case REACHABILITY:
                         if (queryOk.getReachability().getSuccess()) {
                             query.setQueryState(QueryState.SUCCESSFUL);
+                            if(value.toString().contains("true")){
```

Comment:
Indeholder value ikke en bool for at indikere succes i stedet for at konvertere det til en string?